templates/default/layout/html/layout.erb

```diff
@@ -2,6 +2,9 @@
 <html>
   <head>
     <%= erb(:headers) %>
+    <meta name="description" content="The main documentation for the openhab-scripting helper library.">
+    <meta property="og:description" content="The main documentation for the openhab-scripting helper library.">
+    <meta property="og:image" content="https://openhab.github.io/openhab-jruby/main/docs/images/jrubyscripting.png">
```

Comment:
We should add an og:image:alt. "The JRuby logo" sound good?

Reply:
yes that sounds good